### DISCUSSION TITLE
Can't update my action in the marketplace

### DISCUSSION THREAD

**Discussion by skx** [community/community#25834] [Actions]
<p>I put together an action a while back, as you can see there are several releases:</p>
<p><a href="https://github.com/skx/github-action-publish-binaries/releases" rel="nofollow noopener">https://github.com/skx/github-action-publish-binaries/releases</a></p>
<p>However in the marketplace the listing shows a very old release as the most recent:</p>
<p><a href="https://github.com/marketplace/actions/github-action-publish-binaries" rel="nofollow noopener">https://github.com/marketplace/actions/github-action-publish-binaries</a></p>
<p>The documentation suggests that I should check a “Publish to marketplace” checkbox when I’m making a new release, but I see no such option when either I push a new tag, or create a new release via the UI.</p>
<p>What am I missing?</p>

**Comment by skx** [comment, marked as answer]
<p>I looked at an “official” action, <a href="https://github.com/actions/labeler" rel="nofollow noopener">actions/labeler</a> and that provided the clue.</p>
<p>You must now add <code>action.yml</code> inside your repository.  Once that was done I found that I could create a new release, which did then show the “publish to marketplace” text.</p>

**Comment by jakejarvis** [comment]
<p>Thanks for posting your fix, this was driving me crazy!</p>
